text/0000-grid-inventory.md

```diff
@@ -0,0 +1,126 @@
+implementations of Grid Inventory may derive from or extend this
+implementation.
+
+Grid Inventory leverages other extensible Hyperledger Grid components
+including: Grid Product, and Grid Location. Grid Inventory aims to provide
+current inventory quantity & status visibility for each Grid Product and Grid
+Location defined within the Grid Inventory smart contract between parties, as
+well as a summary of relevant inventory events that occurred within a
+time-range.  It includes transactions for generating summary inventory report
+information from external systems of record and recording the results in state.
+
+Grid Inventory also aims to serve as a foundation for future potential Grid
+capabilities that may require inventory related information, such as purchasing
+and logistics.
+
+# Motivation
+[motivation]: #motivation
+
+The Grid Inventory implementation is designed to enable shared visibility of
+summary inventory information for any defined Grid Product and Grid Location
+between participants.  Management of inventory levels is a common concept
+within supply chain solutions and is a key primary business activity related
+to all purchases of physical goods from suppliers, as well as all sales of
+physical goods to customers.  Management of inventory levels is typically
+coordinated very manually and very independently between parties across siloed
+systems and processes.  This siloed and manual coordination of inventory
+between parties can lead to substantial pain-points, including:
+inefficiencies, waste, and poor coordination of inventory levels across
+the supply chain.
+
+This Grid Inventory concept aims to solve these issues by offering a common
+industry solution for vertically integrating inventory information between
+parties existing systems of record.
+
+The Grid Inventory design addresses several use cases, including:
+
+- View Current Inventory Status Data: Viewing current summary inventory
+  quantities and inventory statuses between trading partners for defined Grid
+  Products and Grid Locations.
+
+- View Historical Inventory Activity Data: Viewing historical inventory
+  activity between trading partners for defined Grid Products and Grid
+  Locations.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+TBD - based on discussion.  We are exploring designs based on:
+
+ - GS1’s EDI Inventory Report - Business Message Standard.
+ - GS1's X12 EDI Standards.
+ - GS1's EPCIS Standard.
+ - Other standards as relevant.
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+TBD - based on discussion.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+By leveraging GS1’s EDI Inventory Report standards, only a summary view of
+inventory information can be visible between parties.  Visibility of batch/lot
+or serialized product instance information will not be visible in inventory
+using this initial Grid Inventory design.  In addition, in order for this
```

Comment:
Is this true? I would think we're supporting SGTIN is allowed in the spec, not that orgs would necessarily do it?

Reply:
Hi Dave - Thanks for this comment. The more we look at things, i think it makes more sense to transition away from the EDI inventory report standard (which is used mostly just outside of N. America), more towards the N. America GS1 X12 XRG EDI standards.   I will investigate if they support this.  I believe they will support Batch/Lot - will look to understand if they support SGTIN.